homeassistant/components/comelit/climate.py

```diff
@@ -124,6 +124,7 @@ def _clima(self) -> list[Any]:
     def _api_mode(self) -> str:
         """Return device mode."""
         # Values from API: "O", "L", "U"
+        _LOGGER.warning("Unknown API mode '%s' received", self._clima[2])
```

Comment:
The `_api_mode` property is used by the entity state properties so this warning will still be logged when the entity state properties are accessed. We want to avoid side effects to happen when entity state properties are accessed.

If we still want to keep the warning log, it needs to be moved to an update callback. That will probably require a larger refactor.

Reply:
Removed